hat/assets/js/apps/Iaso/domains/users/hooks/useGetUserRolesDropDown.ts

```diff
@@ -14,8 +14,6 @@ export const useGetUserRolesDropDown = (): UseQueryResult<
         queryFn: () => getRequest('/api/userroles/'),
         snackErrorMsg: MESSAGES.userRolesDropDownError,
         options: {
-            staleTime: 1000 * 60 * 15, // in MS
-            cacheTime: 1000 * 60 * 5,
```

Comment:
This is wrong. You should change the query key instead, so that it is invalidated when user roles are saved

Reply:
Also this hook has a duplicate `useGetUserRolesOptions` in `/domains/userRoles/hooks/requests/useGetUserRoles`, we should keep only one if possible